### PR TITLE
dx-1017 added aborted/failed operations

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/payments-operations.md [hunk 1]
```diff
@@ -95,3 +95,77 @@ Content-Type: application/json
...
+### Operation `failed-payment`
+
+The `failed-payment` operation means that something went wrong during the 
+payment process, the transaction was not authorized, and no further transactions
+can be created if the payment is in this state.
+Under `details` you can see the problem message and under `problems` you can 
+see which problem and the `description` of the problem with the corresponding errorcode.
+ 
+
+{:.code-header}
```
Comment (payex-dev) [inline _includes/payments-operations.md:108]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5763276)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5763276&bid=18865238)

File: _includes/payments-operations.md [hunk 2]
```diff
@@ -5,10 +5,10 @@ and that the payment is completed. Under `details` you can see which card was
 used to complete the payment.
```
Comment (asbjornu) [inline _includes/payments-operations.md:6]:
````suggestion

A `paid-payment` operation looks like the following:

```json
{
   "href": "{{ site.api_url }}/psp/{{ api_resource }}/payments/{{ site.payment_id }}/paid",
   "rel": "paid-payment",
   "method": "GET",
   "contentType": "application/json"
}
```

To inspect the paid payment, you need to perform an HTTP `GET` request
towards the operation's `href` field. An example of how the request and
response look like is given below.

````

File: _includes/payments-operations.md [hunk 3]
```diff
@@ -95,3 +95,77 @@ Content-Type: application/json
...
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+   "problem": {
+       "type": "https://api.payex.com/psp/errordetail/{{ api_resource }}/acquirererror",
```
Comment (asbjornu) [inline _includes/payments-operations.md:127]:
```suggestion
       "type": "{{ site.api_url }}/psp/errordetail/{{ api_resource }}/acquirererror",
```

File: _includes/payments-operations.md [hunk 4]
```diff
@@ -95,3 +95,77 @@ Content-Type: application/json
...
+  }
+}
+```
+
+### Operation `aborted-payment`
+
+The `aborted-payment` operation means that the merchant has aborted the payment 
+before the end user has fulfilled the payment process. You can see this under
+`abortReason` in the response.
+
```
Comment (asbjornu) [inline _includes/payments-operations.md:147]:
````suggestion

An `aborted-payment` operation looks like the following:

```json
{
    "href": "{{ site.api_url }}/psp/creditcard/payments/<paymentId>/aborted",
    "rel": "aborted-payment",
    "method": "GET",
    "contentType": "application/json"
}
```

To inspect why the payment was aborted, you need to perform an HTTP `GET`
request towards the operation's `href` field. An example of how the request and
response looks like is given below.

````

File: _includes/payments-operations.md [hunk 5]
```diff
@@ -95,3 +95,77 @@ Content-Type: application/json
...
 | └─➔&nbsp;`externalNonPaymentToken`| `string`  | Result of external tokenization. This value varies depending on cards, acquirer, customer, etc. For ICA cards, the token comes in response from Swedbank. For Mass Transit(SL) it is populated with PAR if it comes in response from the redeemer (Visa). If not, our own token (Mastercard / Amex).                                                                                                                                                      |
+
+### Operation `failed-payment`
+
+The `failed-payment` operation means that something went wrong during the 
+payment process, the transaction was not authorized, and no further transactions
+can be created if the payment is in this state.
+Under `details` you can see the problem message and under `problems` you can 
+see which problem and the `description` of the problem with the corresponding errorcode.
+
```
Comment (asbjornu) [inline _includes/payments-operations.md:106]:
````suggestion

A `failed-payment` operation looks like the following:

```json
{
   "href": "{{ site.api_url }}/psp/{{ api_resource }}/payments/{{ site.payment_id }}/failed",
   "rel": "failed-payment",
   "method": "GET",
   "contentType": "application/problem+json"
}
```

To inspect why the payment failed, you need to perform an HTTP `GET` request
towards the operation's `href` field.

Under `details` you can see the problem message and under `problems` you can
see which problem and the `description` of the problem with the corresponding
error code.

An example of how the request and response look like is given below.
````

**Pull request merged**